examples/rsp-next-ts/pages/_app.tsx

```diff
@@ -44,7 +44,7 @@ function MyApp({ Component, pageProps }: AppProps) {
             </ActionButton>
           </Flex>
           <View>
-            {/* @ts-ignore */}
+            {/* @ts-ignore - https://github.com/vercel/next.js/issues/49610 */}
```

Comment:
Just a heads up but apparently bumping to [TS5.1.3](https://github.com/vercel/next.js/issues/49610#issuecomment-1576936320) fixed this issue. Fine with keeping this ignore for now anyways since it isn't a RSP issue

Reply:
Yeah but then we have the lodash issue I mentioned in the description 🙃

Reply:
Ahhh, so we want the project's TS version to exactly match the test app's TS version?

Reply:
Ah I can try upgrading just the example to the latest yeah